requirements.txt

```diff
@@ -1,4 +1,4 @@
-ops==2.4.1
+ops
```

Comment:
I think all changes related to unpinning the `ops` version deserve a separate PR.

Reply:
https://github.com/canonical/sdcore-nssf-operator/pull/20

PR created for that.
Once merged, I'll rebase this one.

Reply:
Rebased, now only the issues workflow is contained in this pr.